lib/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { isValidElement, useCallback, useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useCombobox } from 'downshift';
+import { TGenericSizes } from '../Text';
+import { Text } from '../Text';
+import  {Input } from '../Input/index';
+import './Dropdown.scss';
+
+type InputProps = React.ComponentProps<typeof Input>;
+type TProps = {
+    disabled?: boolean;
+    dropdownIcon: React.ReactNode;
+    errorMessage?: InputProps['message'];
+    icon?: React.ReactNode;
+    isRequired?: boolean;
+    label?: InputProps['label'];
+    list: {
+        text?: React.ReactNode;
+        value?: string;
+    }[];
+    listHeight?: Extract<TGenericSizes, 'lg' | 'md' | 'sm'>;
```

Comment:
you don't need to export the type in Text component
```suggestion
    listHeight?: Extract<React.ComponentProps<typeof TGenericSizes>, 'lg' | 'md' | 'sm'>;
```